back/app/services/concerns/json_forms_ideas_overrides.rb

```diff
@@ -122,15 +122,37 @@ def custom_form_topic_ids_to_ui_schema_field field, locale
       type: 'Control',
       label: handle_title(field, locale),
       scope: "#/properties/#{field.key}",
-      options: topics.map{|t| {id: t.id, attributes: {title_multiloc: t.title_multiloc}}}
+      options: {available_topics: topics.map{|t| {id: t.id, attributes: {title_multiloc: t.title_multiloc}}}}
     }
   end
 
-  def custom_form_location_to_ui_schema_field field, locale
+  def custom_form_location_description_to_ui_schema_field field, locale
     {
       type: 'Control',
       label: handle_title(field, locale),
       scope: "#/properties/#{field.key}",
     }
   end
+
+  def custom_form_location_point_geojson_to_json_schema_field field, locale
+    {
+      title: handle_title(field, locale),
```

Comment:
It can be label on the UISchema or title in the schema. I'm going to undo this change, it's out of scope, but I've removed rendering the label on the parent multiloc layout and transferred the string here. I have to rehaul the multiloc component anyways because validation doesn't work as is.